delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ViewRef.scala

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model
+
+import cats.Order
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+
+/**
+  * A view reference.
+  *
+  * @param project the view parent project
+  * @param viewId  the view id
+  */
+final case class ViewRef(project: ProjectRef, viewId: Iri)
+
+object ViewRef {
+
+  // required for NonEmptySet
+  // sort by project first and then by view id
+  implicit final val viewRefOrder: Order[ViewRef] = Order.from { (fst, snd) =>
```

Comment:
More readable and simple:
```scala
implicit val viewRefOrder: Ordering[ViewRef] = 
   Ordering.by{case ViewRef(project, viewId) => (project,viewId)}
```

We will need an `Ordering` for `ProjectRef` and `Iri`, but that's fine, since it will be used in multiple places (ordering on `ProjectRef` might be used in resolvers too)